ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -0,0 +1,289 @@
+  ASYNC_INDICATORS_TO_QUEUE: 30000
+  BANK_ACCOUNT_NUMBER: "{{ localsettings_private.BANK_ACCOUNT_NUMBER }}"
+  BANK_ADDRESS: { 'first_line': "1 Citizens Drive", 'city': "Riverside", 'region': "RI", 'postal_code': "02915" }
+  BANK_NAME: "RBS Citizens N.A."
+  BANK_SWIFT_CODE: 'CTZIUS33'
+  BANK_ROUTING_NUMBER_ACH: "{{ localsettings_private.BANK_ROUTING_NUMBER_ACH }}"
+  BANK_ROUTING_NUMBER_WIRE: "{{ localsettings_private.BANK_ROUTING_NUMBER_WIRE }}"
+  BITLY_APIKEY: "{{ localsettings_private.BITLY_APIKEY }}"
+  BITLY_LOGIN: "{{ localsettings_private.BITLY_LOGIN }}"
+  BOOKKEEPER_CONTACT_EMAILS: "{{ localsettings_private.BOOKKEEPER_CONTACT_EMAILS }}"
+  BROKER_URL: 'amqp://{{ secrets.AMQP_USER }}:{{ secrets.AMQP_PASSWORD }}@{{ AMQP_HOST }}:5672/{{ AMQP_NAME }}'
+  CELERY_FLOWER_URL: "http://{{ groups.celery.1 }}:5555"
+  CELERY_PERIODIC_QUEUE: 'celery_periodic'
+  CELERY_REMINDER_CASE_UPDATE_QUEUE: 'reminder_case_update_queue'
+  CELERY_REMINDER_RULE_QUEUE: 'reminder_rule_queue'
+  CELERY_REPEAT_RECORD_QUEUE: 'repeat_record_queue'
+  CELERY_RESULT_BACKEND: 'djcelery.backends.database:DatabaseBackend'
+#  COUCH_CACHE_DOCS:
+#  COUCH_CACHE_VIEWS:
+  COUCH_USERNAME: "{{ localsettings_private.COUCH_USERNAME }}"
+  COUCH_PASSWORD: "{{ localsettings_private.COUCH_PASSWORD }}"
+  DEPLOY_MACHINE_NAME: "{{ inventory_hostname }}"
+  DROPBOX_APP_NAME: 'CommCareHQFiles'
+  DROPBOX_KEY: "{{ localsettings_private.DROPBOX_KEY }}"
+  DROPBOX_SECRET: "{{ localsettings_private.DROPBOX_SECRET }}"
+  ELASTICSEARCH_HOST: "{{ groups.elasticsearch.0 }}"
+  ELASTICSEARCH_PORT: '9200'
+  EMAIL_LOGIN: "{{ localsettings_private.EMAIL_LOGIN }}"
+  EMAIL_PASSWORD: "{{ localsettings_private.EMAIL_PASSWORD }}"
+  EMAIL_SMTP_HOST: 'localhost'
+  EMAIL_SMTP_PORT: 25
+  EMAIL_USE_TLS: no
+  ENABLE_DRACONIAN_SECURITY_FEATURES: yes
+  FORMPLAYER_URL: "/formplayer"
+  GMAPS_API_KEY: "{{ localsettings_private.GMAPS_API_KEY }}"
+  GOOGLE_ANALYTICS_API_ID: '{{ localsettings_private.GOOGLE_ANALYTICS_API_ID }}'
+  HQ_INSTANCE: 'enikshay'
+  HQ_PRIVATE_KEY: "{{ localsettings_private.HQ_PRIVATE_KEY }}"
+  HUBSPOT_API_ID: '{{ localsettings_private.HUBSPOT_API_ID }}'
+  HUBSPOT_API_KEY: '{{ localsettings_private.HUBSPOT_API_KEY }}'
+#  ICDS_UCR_DATABASE_ALIAS: icds-ucr
+#  ICDS_UCR_DATABASE_NAME: icds-ucr
+  INACTIVITY_TIMEOUT: 20160
+  INVOICE_FROM_ADDRESS:
+    'name': "Dimagi, Inc."
+    'first_line': "585 Massachusetts Ave"
+    'city': "Cambridge"
+    'region': "MA"
+    'postal_code': "02139"
+    'country': "US"
+    'phone_number': "(617) 649-2214"
+    'email': "accounts@dimagi.com"
+    'website': "http://www.dimagi.com"
+  INVOICE_PREFIX: Enikshay-
+  INVOICE_STARTING_NUMBER: 5000
+  JAR_KEY_ALIAS: javarosakey
+  JAR_KEY_PASS: "{{ localsettings_private.JAR_KEY_PASS }}"
+  JAR_STORE_PASS: "{{ localsettings_private.JAR_STORE_PASS }}"
+  J2ME_ADDRESS: "{{ J2ME_SITE_HOST }}"
+  KAFKA_URL: "{{ groups.kafka.0 }}"
+  KISSMETRICS_KEY: "{{ localsettings_private.KISSMETRICS_KEY }}"
```

Comment:
@snopoke Currently we have just kept everything. It surely looked like somethings won't be needed for this set up but didn't knew it off hand. Thanks for flagging these. Will update it.